src/server.ts

```diff
@@ -16,13 +16,13 @@ app.use(express.json())
 app.post('/api/costs', async (req, res) => {
   const newCosts = req.body
   await getCollection().insertOne(newCosts)
-  res.status(200).send('Newcost was added')
+  res.status(200).send('Newcost were added')
```

Comment:
"New costs" or "New cost'

Reply:
thank you for your code review.. yes it´s better New cost, because it is always only one